### PR TITLE
Feature/nwps 833 right hand image component

### PR DESCRIPTION
**Pull request by dawntaroni**
Sidebar image styling

### CODE REVIEW

**Comment by dawntaroni** [comment]
https://hee-tis.atlassian.net/browse/NWPS-833

File: app/views/lks/content.html [hunk 1]
```diff
@@ -86,7 +86,12 @@ <h2>Copyright Act & 'Library Privilege'</h2>
         </section>
       </div>
 
-      <div class="nhsuk-grid-column-one-third">
+      <div class="nhsuk-grid-column-one-third">     
+        
+        <div class="nhsuk-sidebar-image">
+          <img src="/images/placeholders/dental-region-map.png" alt="Dental region map" />
+        </div>
```
Comment (Raq-HEE) [inline app/views/lks/content.html:93]: Probably overkill for this, but would it be possible to have this rendered using nunjucks to keep consistent with other front end components? Not sure how familiar you are with nunjucks, so may be a nice simple one to play around with.

**Review by Raq-HEE — COMMENTED** [review]
Noticed as you reduce the size of the browser width, the image does not scale down, so ends up overlapping the image border, and suspect it may be causing the horizontal scroll also:
image
Scaling the image might be tricky, as when you jump to tablet/mobile view the image would be scaled up - which might be fine - but worth checking with Andy/Will on the options.

**Comment by Raq-HEE** [comment]
Would it be possible to include a readme. You can look at another component as a guide. It might be worth adding the required image dimensions in the readme for reference also.

**Comment by dawntaroni** [comment]
@Raq-HEE I've added a readme file, swapped it to a nunjuck component and fixes the issues for styling. Ready for review again

**Pull request merged**